### PR TITLE
create default user

### PR DESCRIPTION
**Pull request by smacker**
fix: #11

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -49,6 +49,11 @@ services:
     image: srcd/superset:latest
     restart: unless-stopped
     environment:
+      ADMIN_LOGIN: admin
+      ADMIN_FIRST_NAME: admin
+      ADMIN_LAST_NAME: admin
+      ADMIN_EMAIL: admin@no-email.org
```
Comment (carlosms) [inline docker-compose.yml]: I'm not familiar with `no-email.org`, maybe it's safer to use `admin@example.com`, since it's owned by IANA

**Comment by smacker** [comment]
1. rebased
2. email changed
3. docs updated

**Comment by smacker** [comment]
Tienes razón! I'll wait for that PR to be merged and then update preserving the init script.

**Comment by smacker** [comment]
Rebased on master. Don't forget to update `~/.srcd/docker-compose.yml` with the new yml file for testing.

**Pull request merged**